ctapipe/io/dl1eventsource.py

```diff
@@ -105,6 +105,8 @@ def is_compatible(file_path):
                 return False
             if metadata["CTA PRODUCT DESCRIPTION"] != "DL1 Data Product":
```

Comment:
This and the previous check can be simplified to `metadata.get('CTA PRODUCT DESCRIPTION') != "DL1 Data Product"